src/services/schemas/properties/SchemaPropertyService.ts

```diff
@@ -99,6 +100,10 @@ export abstract class SchemaPropertyService {
     const defaults = getSchemaPropertyDefaultValidators(this.property, required)
     const validators = fieldRules(title, ...this.validators)
 
+    if (this.componentIs(JsonInput)) {
+      validators.push(isJson(title))
```

Comment:
In case anyone else ends up wondering, `isJson` gracefully handles null values:
```
  if (isNullish(value) || isEmptyString(value)) {
    return true
  }
```